pom.xml

```diff
@@ -1101,5 +1101,61 @@
         </plugins>
       </build>
     </profile>
+    <!-- Profiles preconfiguring different Jenkins versions. -->
+    <!-- LTS lines are used. -->
+    <profile>
+      <id>jenkins-642</id>
+      <properties>
+        <jenkins.version>1.642.1</jenkins.version>
+        <hpi-plugin.version>1.115</hpi-plugin.version>
+        <stapler-plugin.version>1.17</stapler-plugin.version>
+        <java.level>7</java.level>
+      </properties>
+    </profile>
+    <profile>
+      <id>jenkins-625</id>
+      <properties>
+        <jenkins.version>1.625.3</jenkins.version>
+        <hpi-plugin.version>1.106</hpi-plugin.version>
+        <stapler-plugin.version>1.17</stapler-plugin.version>
+        <java.level>7</java.level>
+      </properties>
+    </profile>
+    <profile>
+      <id>jenkins-609</id>
+      <properties>
+        <jenkins.version>1.609.3</jenkins.version>
+        <hpi-plugin.version>1.106</hpi-plugin.version>
+        <stapler-plugin.version>1.17</stapler-plugin.version>
+        <java.level>6</java.level>
+      </properties>
+    </profile>
+    <profile>
+      <id>jenkins-596</id>
+      <properties>
+        <jenkins.version>1.596.3</jenkins.version>
+        <hpi-plugin.version>1.106</hpi-plugin.version>
+        <stapler-plugin.version>1.17</stapler-plugin.version>
+        <java.level>6</java.level>
+      </properties>
+    </profile>
+    <profile>
+      <id>jenkins-580</id>
+      <properties>
+        <jenkins.version>1.580.4</jenkins.version>
+        <hpi-plugin.version>1.106</hpi-plugin.version>
+        <stapler-plugin.version>1.17</stapler-plugin.version>
+        <java.level>6</java.level>
+      </properties>
+    </profile>
+    <profile>
+      <id>jenkins-565</id>
+      <properties>
+        <jenkins.version>1.565.3</jenkins.version>
```

Comment:
Address. For profiles >= 1.580.1 I've left the default instead of setting it explicitly to avoid having to (and forgetting to) change the value in different places as new JTH versions are released. When a JTH version appears that requires an specific version the exceptions will be added.